modules/node_modules/@ncigdc/containers/RepositoryPage.js

```diff
@@ -120,42 +132,35 @@ export const RepositoryPageQuery = {
     files_size: null,
     files_sort: null,
     filters: null,
-    idAutocompleteCases: null,
-    runAutocompleteCases: false,
-    idAutocompleteFiles: null,
-    runAutocompleteFiles: false,
+    autocompleteTypes: [],
+    idAutocomplete: null,
+    runAutocomplete: false,
   },
   fragments: {
-    /*
-    autocomplete: () => Relay.QL`
-      fragment on Repository {
-        cases {
-          hits(first: 5, id_autocomplete: $idAutocompleteCases) {
-            edges @include(if: $runAutocompleteCases) {
-              node {
-                case_id
-                project {
-                  project_id
-                }
+    viewer: () => Relay.QL`
+      fragment on Root {
+        autocomplete_case: query (query: $idAutocomplete types: ["case"]) @include(if: $runAutocomplete) {
```

Comment:
you need to split up id and run into case and file right? or you will call both end points